src/_codux/boards/add-to-cart/add-to-cart.board.tsx

```diff
@@ -1,13 +1,15 @@
 import { createBoard } from '@wixc3/react-board';
-import AddToCart from '../../../components/AddToCart';
+import { AddToCart } from '../../../components/AddToCart';
 
 export default createBoard({
-   name: 'AddToCart',
-   Board: () => <AddToCart />,
-   isSnippet: true,
-   environmentProps: {
-      canvasWidth: 789,
-      windowWidth: 1090,
-      canvasHeight: 82,
-   },
+    name: 'AddToCart',
+    Board: () => <AddToCart setCartCount={function(count: number | ((prevCartCount: number) => number)): void {
```

Comment:
You can pass a dummy function as the functionality itself isn't always easy to replicate on boards.